shuup/campaigns/admin_module/sections.py

```diff
@@ -25,14 +27,17 @@ def visible_for_object(cls, product, request=None):
     @classmethod
     def get_context_data(cls, product, request=None):
         ctx = {}
-        shop = request.shop
+        shop = get_shop(request)
         try:
             shop_product = product.get_shop_instance(shop)
+            basket_campaigns = BasketCampaign.get_for_product(shop_product)
+            user = request.user
+            if not (getattr(user, 'is_superuser', False) and shop.staff_members.filter(id=user.pk).first()):
```

Comment:
what is the point here with this check? Is this right? You only check whether the user is staff member if he is a superuser? Does this make sense?

Reply:
Should be or I guess.

Reply:
"I guess" :yum: super confident haha